src/components/storage-browser.js

```diff
@@ -323,6 +324,14 @@ class StorageBrowser extends React.Component {
       }
     }
 
+    if (requiredBytes > LIMIT_PER_STORAGE_UPLOAD_IN_BYTES) {
+      message.error(t('uploadLimitExceeded', {
+        uploadSize: prettyBytes(requiredBytes, { locale: this.props.uiLocale }),
+        uploadLimit: prettyBytes(LIMIT_PER_STORAGE_UPLOAD_IN_BYTES, { locale: this.props.uiLocale })
+      }));
+      return;
```

Comment:
Maybe let's also set the same limit in the middleware of the controller so the server doesn't accept more than `LIMIT_PER_STORAGE_UPLOAD_IN_BYTES`, what do you think?

Reply:
Of course, added it on the server side too, thanks!

Reply:
Cool thanks :+1: